ESSArch_PP/frontend/static/frontend/views/remove-order-modal.html

```diff
@@ -0,0 +1,10 @@
+<div class="enter-profile-name">
```

Comment:
The naming of this class doesn't fit well with how it is used. The class should either be replaced here with a new one or be renamed.